app/src/pages/explorer/chart/datavoyager/data-voyager-page.html

```diff
@@ -4,13 +4,13 @@
         class="utility-gridicon"
         v-if="!loading"
       >
-        <div> 
+        <div>
             <md-button
                 class="md-icon-button"
                 @click.native.prevent="navBack"
```

Comment:
button seems to have stopped working

Reply:
behavior was undefined for a new chart. I changed navBack to just do router.back.